tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -569,8 +586,8 @@ def docomplicated(suptitle=None):
 # the difference between the red ``pos`` box and the green ``ax`` box
 # is set by the size of the decorations around the Axes.
 #
-# In the code, this is accomplished by the entries in `.do_constrained_layout`
-# like::
+# In the code, this is accomplished by the entries in
+# ``.do_constrained_layout`` like::
```

Comment:
Either no leading dot (and possibly trailing parentheses) or a proper link.

Reply:
Cannot link to it, because that is a method from a private module which does not occur inside the docs. One may ask in general why all those methods are exposed here, while not being documented, but there might be a reason, so I'm not cutting this part.